src/jit/regset.cpp

```diff
@@ -1384,7 +1393,7 @@ void RegTracker::rsTrackRegLclVar(regNumber reg, unsigned var)
 
     regSet->rsSetRegsModified(genRegMask(reg));
 
-#if REDUNDANT_LOAD
+#ifdef LEGACY_BACKEND
```

Comment:
`REDUNDANT_LOAD` is always 1 so I removed `#if REDUNDANT_LOAD`, there are too many ifdefs already. I kept the define itself (in `jit.h`) because it's used in `codegenlegacy.cpp` and it probably doesn't make much sense to cleanup that.